UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.cs

```diff
@@ -1192,12 +1192,13 @@ private void AirTimeChecks()
 				var floating = IsFloating();
 				if (floating == false)
 				{
-					AppliedFriction(DEFAULT_SLIDE_FRICTION);
+					AppliedFriction(DEFAULT_Friction);
```

Comment:
oops this is my bad should be DEFAULT_SLIDE_FRICTION xD

Reply:
done